compiler/qsc_lowerer/src/lib.rs

```diff
@@ -153,8 +157,6 @@ impl Lowerer {
         }
 
         fir_package.entry = entry;
-
-        qsc_fir::validate::validate(fir_package);
```

Comment:
Hmm, this is an unfortunate removal, but has no effect on behavior since validation is just for checking and catching compiler bugs. I see why it was needed, since validate now requires the store as a parameter, and we don't have one here. It's highly unlikely we'll introduce bugs of the kind this validate would check (including references to ids that don't get added to the corresponding maps). I'll resolve this comment, just wanted note the change.